.gitlab/tests.yml

```diff
@@ -13,6 +13,11 @@ variables:
   # DEV: we have a larger pool of amd64 runners, prefer that over arm64
   tags: [ "arch:amd64" ]
   timeout: 20m
+  variables:
+    KUBERNETES_CPU_REQUEST: 12
```

Comment:
would it be better to have a lower request than limit?

most of our jobs aren't going to use 12 CPUs anyways, majority of them are single threaded.

better to have request 1/2 and limit of 12?

same with memory, request of 3/4 and limit of 16?

Reply:
nvm, based on how the limits are enforced, requests must equal the limit.

I still wonder if this is too aggressive to apply to all of our testrunner jobs, should we be more targeted?